### DISCUSSION TITLE
pages/UA-PNP/writer_resources/writer_resources_doc/topics/newwriters

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#42] [Q&A]
# pages/UA-PNP/writer_resources/writer_resources_doc/topics/newwriters

New writers require guidance before they start to write product documentation.

https://github01.hclpnp.com/pages/UA-PNP/writer_resources/writer_resources_doc/topics/newwriters.html

**Comment by melanie-feb** [comment]
test comment

**Reply by melanie-feb** [reply]
reply
